.travis.yml

```diff
@@ -17,7 +17,7 @@ install:
   - pip install pytest==5.2.0 pytest-cov==2.5.1 pytest-pep8 coveralls
 
 script:
-  - python -m pytest caliban_toolbox tests
+  - python -m pytest caliban_toolbox
```

Comment:
Why not include a pep8 and coverage test here too?

`python -m pytest --pep8 --cov=caliban_toolbox caliban_toolbox`